default/python/universe_generation/util.py

```diff
@@ -43,5 +45,12 @@ def report_error(msg):
     print(msg, file=sys.stderr)
 
 
+def iterate_pairs(lst: Iterable[int], lst2: Iterable[int]) -> Iterable[Tuple[int, int]]:
```

Comment:
`unique_product`

Reply:
I don't like when variables has numbers in their names.  lst, lst2 -> first, second

Reply:
Fixed.